src/UCTSearch.cpp

```diff
@@ -67,6 +66,7 @@ class OutputAnalysisData {
     std::string m_move;
     int m_visits;
     int m_winrate;
+    std::string m_N_num; // policy prior
```

Comment:
Rename this to m_policy_prior please, and store it as a float. (Do the conversion at display time)